NAMESPACE

```diff
@@ -12,11 +12,13 @@ S3method(type,stringr_fixed)
 S3method(type,stringr_regex)
 export("%>%")
 export("str_sub<-")
+export(StrToPascal)
```

Comment:
Now that I see them, I don't think it's worth having the camel case variants; it feels a bit too clever.